openedx/core/djangoapps/course_groups/models.py

```diff
@@ -135,6 +142,10 @@ def save(self, *args, **kwargs):
         if not success:
             raise IntegrityError("Unable to save membership after {} tries, aborting.".format(max_retries))
 
+    @receiver(pre_delete)
```

Comment:
It looks like this is causing test failures because it's too general. I feel like you should be able to scope this to only listen to particular models.